src/promptflow-devkit/promptflow/_sdk/_constants.py

```diff
@@ -474,6 +474,8 @@ class Local2Cloud:
     ASSET_NAME_DEBUG_INFO = "debug_info"
     ASSET_NAME_FLOW_OUTPUTS = "flow_outputs"
     EXECUTION_LOG = "logs/azureml/executionlogs.txt"
+    # instance_results.jsonl contains the inputs and outputs of all lines
+    FLOW_INSTANCE_RESULTS_FILE_NAME = "instance_results.jsonl"
```

Comment:
one silly questoin, what is instance results? may add to the pr desc.

Reply:
updated in description